index.js

```diff
@@ -1,17 +1,49 @@
 class SortedList {
-  constructor() {}
+  constructor(items, lenght) {
+    this.items = [];
+    this.length = this.items.length;
+    this.lenght === 0;
 
-  add(item) {}
 
-  get(pos) {}
+  }
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
+  get(pos) {
+    if (pos > this.length) {
+      throw new Error('OutOfBounds');
+    } else {
+      return this.items[pos];
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items[this.items.length - 1];
+    }
+  }
+  min() {
+    if (this.items.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.items[0];
+    }
+  }
+  sum() {
+    return this.items.reduce((suma, i) => suma + i);
```

Comment:
No se por que decis que no pudiste resolver `sum()`, yo lo veo bastante bien, te falta el cero al final

`return this.items.reduce((suma, i) => suma + i, 0)`

Reply:
gracias Tomas!